### ISSUE TITLE
Align the submission form and database with UN CRPD articles and objectives related to technology

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wai-policies-prototype#205]
Rebecca's feedback (2 of 2):

I understand that the intent of this form/table is to gather information about laws and policies specific to Web Accessibility. Looking towards the future, it would be interesting to tie in/align questions with the CRPD articles and objectives related to technology. This would be information that countries should already be looking for in writing their reports to the Committee on the Rights of Persons with Disabilities with respect to their implementation of the CRPD.
 
For example, in **Article 4, General Obligations**
 
Countries… undertake/promote… (4.1.g)

- research and development of new technologies
- the availability and use of new technologies

Undertake/promote research and development and promote the availability and use of… (4.1.g)

- ICT suitable for persons with disabilities
- mobility aids… suitable for persons with disabilities
- devices and assistive technologies, suitable for persons with disabilities

Give priority to… (4.1.g)

- technologies at an affordable cost

Provide accessible information to persons with disabilities about… (4.1.h)

- mobility aids
- devices and assistive technologies
- new technologies
 
### Article 9: Accessibility:

Countries… take appropriate measures to ensure persons with disabilities access, on an equal basis with others to… (9.1)

- information and communications technologies and systems

these measures shall apply to emergency services (9.1.b)

take appropriate measures to promote access for persons with disabilities to… (9.2)

- new information and communications technologies systems
- the Internet
 
### Article 21: Freedom of Expression and Access to Information

Countries… take appropriate measures to ensure that… (21)

- persons with disabilities can exercise the right to freedom of expression and opinion
- provide information intended for the general public to persons with disabilities… (21.a)

in accessible formats and technologies

- appropriate to different kinds of disabilities
- without additional costs

encourage the mass media to… (21.d)

- make their services available to persons with disabilities

encourage… providers of information through the Internet to… (21.d)

- make their services available to persons with disabilities
 
### Article 30: Participation in Political and Public Life

> Countries… facilitate the use of assistive and new technologies where appropriate… to protect the right of persons with disabilities to… (29.a.ii)
> 
> - vote by secret ballot
> - stand for elections.

 
Gathering references to laws/policies which clearly demonstrate/support these indicators could be really powerful in understanding the implementation and impact of the CRPD.

I received this feedback about the Web Accessibility Policy submission form from Rebecca Sheffield --
 https://www.linkedin.com/in/rebeccasheffieldphd/

**Comment by yatil** [comment]
I like the idea, the question is how to gather that data/progress. Is this published by the UN? We don’t want to interpret the laws/policies ourselves. (I also think it would be a different, but related resource.)

**Comment by maryjom** [comment]
I only see a potential expansion to include all types of ICT in a later version of the policies page so it doesn't simply focus on just web content. This is because WCAG is being used by policy makers in for non-web software, non-web documentation.  Since this is a W3C resource, we have to limit its scope simply to that.

**Comment by mitchellevan** [comment]
Thanks for the questions and comments. I've asked Rebecca Sheffield to respond.

**Comment by RebeccaMSheffield** [comment]
hi all - I'm new to github as of today, so apologies if the links and such in this post are not formatted correctly

The United Nation's Office of the High Commissioner for Human Rights posts all the country reports on implementation of the UN Convention on the Rights of Persons with Disabilities here: [http://tbinternet.ohchr.org/_layouts/treatybodyexternal/TBSearch.aspx?Lang=en&TreatyID=4&DocTypeID=29](http://tbinternet.ohchr.org/_layouts/treatybodyexternal/TBSearch.aspx?Lang=en&TreatyID=4&DocTypeID=29) (countries' initial reports)
[http://tbinternet.ohchr.org/_layouts/treatybodyexternal/TBSearch.aspx?Lang=en&TreatyID=4&DocTypeID=18](http://tbinternet.ohchr.org/_layouts/treatybodyexternal/TBSearch.aspx?Lang=en&TreatyID=4&DocTypeID=18) (committee's responses to initial reports)
[http://tbinternet.ohchr.org/_layouts/treatybodyexternal/TBSearch.aspx?Lang=en&TreatyID=4&DocTypeID=22](http://tbinternet.ohchr.org/_layouts/treatybodyexternal/TBSearch.aspx?Lang=en&TreatyID=4&DocTypeID=22) (countries' replies to the responses)
[http://tbinternet.ohchr.org/_layouts/treatybodyexternal/TBSearch.aspx?Lang=en&TreatyID=4&DocTypeID=5 ](http://tbinternet.ohchr.org/_layouts/treatybodyexternal/TBSearch.aspx?Lang=en&TreatyID=4&DocTypeID=5) (committee's concluding observations)

Initially there was no standardization of the content required in these reports, so finding evidence of specific indicators within each country report became quite a daunting task. I believe that there is a push towards a standard/template for country reports, which will make it easier to skim the reports for evidence of policy changes around a specific topic (such as web accessibility). 

I understand that the focus of this database is W3C resource; hopefully the UNCRPD reports are helpful on that front; and I would love to work towards a related/separate database that uses the implementation reports to document more broadly any evidence of digital accessibility and other measurable indicators for countries' implementation of the CRPD.

**Comment by sharronrush** [comment]
Agree with @maryjom that we cannot change the structure of this database to meet UN policies. Limiting scope to W3C resource statement is correct. Can confirm at the March 2018 F2F of EOWG

Suggest closing this issue.

**Comment by iamjolly** [comment]
Thank you @RebeccaMSheffield for your input. This is interesting and could be valuable to connect this resource in some way with the UNCRPD. We are discussing different possibilities and will have more thoughts and/or questions to share soon. Thanks again for these ideas!

**Comment by nitedog** [comment]
I like the idea of adding a small section (just a sentence or two), possibly below the table, linking to the CRPD and state parties' reports for further information. This can be useful in several ways, as @RebeccaMSheffield indicates:

1. If you don't find the information about your country here, then you may be able to find it in the CRPD reports;
2. The CRPD reports can give you broader information about the context of these policies, and other related policies;
3. Establishing this link could indeed motivate some people, like DPOs, to contribute information on this W3C list.

**Comment by iamjolly** [comment]
I can take this on. Is there any specific language you are thinking of as of now @nitedog?

**Comment by slhenry** [comment]
(Added Chairs Brent & Sharron so they can help manage it through EOWG. :-)

**Comment by RebeccaMSheffield** [comment]
Suggestion (feel free to use as much/as little as you'd like): 

Article 9 of the UNCRPD relates to the W3C Laws & Policies resource. It requires states parties to take measures to ensure equal access to information and communications technologies and systems, including to:
- "develop, promulgate and monitor the implementation of minimum standards" for accessibility of public services and facilities,
- "promote access for persons with disabilities to new information and communications technologies and systems, including the Internet," and 
- "promote the design, development, production and distribution of accessible information and communications technologies and systems." 
Accessibility of information and communications technologies is also addressed in Articles 4, 21, and 29 of the UNCRPD. If you find information about web accessibility laws or policies in UNCRPD states parties' reports or other documents that should be included here, please let us know by opening a new issue in GitHub or by emailing wai-eo-editors@w3.org.

**Comment by sharronrush** [comment]
+1 to Rebecca's suggestion. This is longer than we wanted but the bulleted presentation and clarity of the prose makes it quite readable. I also appreciate how the importance of digital accessibility to UNCRPD is emphasized and contributions are encouraged.

**Comment by maryjom** [comment]
I don't see UNCRPD states looking to our page to figure out what to submit in the states' parties reports.  However, they could submit new regulations they have on the books to list on our page using our submission form.  In addition, I have not been monitoring UNCRPD reports as a source of information for this page because, as mentioned, they are all very differently written and difficult to wade through to find any specific regulatory information. I've been simply doing searches on the internet for articles listing new regulations and by receiving submissions from persons in the various countries that know what regulations apply.

As for what would potentially be added - IMO the UNCRPD and what it requires is not the focus of the W3C page. I do agree that it is definitely a driver of new regulations for accessibility not only of the web but other ICT and deserves mention but not a lot of real-estate on this specific W3C resource.

Therefore, I think we should keep this very minimal, as there were always concerns about too much extra text on this page - we went round and round with that during its design last year. I think if we could simply list the articles that relate in a short statement and then point to the UNCRPD list of articles so users can quickly access them through the available links if they want to read them.  

I'm also not sure where this should actually be put on the page. If it's added to the top of the summary page in the introduction, then there's already information and text for submitting updates or additions via the submission form. I think it might be confusing adding a different sentence with different instructions saying to open a new issue in GitHub or email wai-eo-editors.  We are trying to get away from the more manual submission.

So I prefer something similar to Robert's shorter prose with tweaks. How about this?

The United Nations Convention on the Rights of Persons with Disabilities [UNCRPD] (https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities/article-9-accessibility.html) relates to the W3C Laws & Policies resource. It requires states parties to take measures to ensure equal access to information and communications technologies and systems. Refer to Articles 4, 9, 21, and 29 for more details.
